.github/workflows/test-runner-integration.yml

```
@@ -0,0 +1,55 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Run Workspace Manager Integration Tests
+
+on:
+  push:
+    paths-ignore:
+      - 'README.md'
+      - 'local-dev/**'
+
+jobs:
+  test-runner-perf:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up AdoptOpenJDK 11
+        uses: joschi/setup-jdk@v2
+        with:
+          java-version: 11
+      - name: Get Vault token
+        id: vault-token-step
+        env:
+          VAULT_ADDR: https://clotho.broadinstitute.org:8200
+        run: |
+          VAULT_TOKEN=$(docker run --rm --cap-add IPC_LOCK \
+            -e "VAULT_ADDR=${VAULT_ADDR}" \
+            vault:1.1.0 \
+            vault write -field token \
+              auth/approle/login role_id=${{ secrets.VAULT_APPROLE_ROLE_ID }} \
+              secret_id=${{ secrets.VAULT_APPROLE_SECRET_ID }})
+          echo ::set-output name=vault-token::$VAULT_TOKEN
+          echo ::add-mask::$VAULT_TOKEN
+      - name: "Configure Test Runner"
+        run: |
+          cd ${GITHUB_WORKSPACE}/${workingDir}/workspace-manager-clienttests
+          chmod +x render-config.sh
+          ./render-config.sh ${{ steps.vault-token-step.outputs.vault-token }}
+          chmod +r rendered
+      - name: Cache Gradle packages
+        uses: actions/cache@v2
+        with:
+          path: |
+            .gradle/caches
+            .gradle/wrapper
+          key: v1-${{ runner.os }}-gradle-${{ hashfiles('**/gradle-wrapper.properties') }}-${{ hashFiles('**/*.gradle') }}
+          restore-keys: v1-${{ runner.os }}-gradle-${{ hashfiles('**/gradle-wrapper.properties') }}
+      - name: Run GetWorkspace tests using test runner library
+        run: |
+          cd ${GITHUB_WORKSPACE}/${workingDir}/workspace-manager-client
+          ../gradlew clean
+          ../gradlew build
+          cd ${GITHUB_WORKSPACE}/${workingDir}/workspace-manager-clienttests
```

Is this supposed to just build the client? This will build everything. We also don't need to run clean before. 

```suggestion
          ./gradlew :workspace-manager-client:build
          cd workspace-manager-clienttests
```